framework/dbt/basic/macros/overrides.sql

```diff
@@ -1,116 +0,0 @@
-{% macro default__reset_csv_table(model, full_refresh, old_relation, agate_table) %}
-    {% set sql = "" %}
-    {% if full_refresh %}
-        {{ adapter.drop_relation(old_relation) }}
-        {% set sql = create_csv_table(model, agate_table) %}
-    {% else %}
-        {{ adapter.truncate_relation(old_relation) }}
-        {% set sql = "delete from " ~ old_relation %}
-    {% endif %}
-
-    {{ return(sql) }}
-{% endmacro %}
-
-{% macro generate_schema_name(custom_schema_name, node) -%}
-
-    {%- set default_schema = target.schema -%}
-    {%- if custom_schema_name is none -%}
-
-        {{ default_schema }}
-
-    {%- else -%}
-
-        {{ custom_schema_name | trim }}
-
-    {%- endif -%}
-
-{%- endmacro %}
-
-{% macro postgres__create_schema(relation) -%}
-  {%- call statement('create_schema') -%}
-    /* schemas are not created in CrateDB */
-    DROP TABLE IF EXISTS thisschemadefinitelydoesnotexits.thiswouldnotexist
-    /* but we need to run something to not have just EOF */
-  {% endcall %}
-{% endmacro %}
-
-{% macro postgres__create_table_as(temporary, relation, sql) -%}
-  {%- set unlogged = config.get('unlogged', default=false) -%}
-  {%- set sql_header = config.get('sql_header', none) -%}
-
-  {{ sql_header if sql_header is not none }}
-
-  create  table {{ relation }}
-  as (
-    {{ sql|replace('"crate".', "") }}
-  );
-{%- endmacro %}
-
-{% macro postgres__drop_schema(relation) -%}
-  {% if relation.database -%}
-    {{ adapter.verify_database(relation.database) }}
-  {%- endif -%}
-  {%- call statement('drop_schema') -%}
-    /* schemas are not dropped in CrateDB */
-  {%- endcall -%}
-{% endmacro %}
-
-{% macro default__drop_relation(relation) -%}
-  {% call statement('drop_relation', auto_begin=False) -%}
-    drop {{ relation.type }} if exists "{{ relation.schema }}"."{{ relation.identifier }}"
-  {%- endcall %}
-{% endmacro %}
-
-{% macro default__drop_schema(relation) -%}
-  {%- call statement('drop_schema') -%}
-    /* schemas are not dropped in CrateDB */
-  {% endcall %}
-{% endmacro %}
-
-{% macro default__create_view_as(relation, sql) -%}
-  {%- set sql_header = config.get('sql_header', none) -%}
-
-  {{ sql_header if sql_header is not none }}
-  create view "{{ relation.schema }}"."{{ relation.identifier }}" as 
-    {{ sql|replace('"crate".', "") }}
-  ;
-{%- endmacro %}
-
```

Comment:
I guess those are being covered well by dbt-cratedb2 now, in turn building upon dbt-postgres. The new locations of corresponding CrateDB overrides is here:

- https://github.com/crate/dbt-cratedb2/blob/v0.0.2/dbt/include/cratedb/macros/adapters.sql
- https://github.com/crate/dbt-cratedb2/blob/v0.0.2/dbt/include/cratedb/macros/relations.sql

Please check if you think something is missing here.